partcad-cli/src/partcad_cli/click/command.py

```diff
@@ -3,10 +3,19 @@
 import partcad as pc
 import coloredlogs
 import logging
+import sentry_sdk
+import rich_click as click
+import yaml
+import partcad as pc
+import coloredlogs
+import logging
+
+from partcad.logging_ansi_terminal import init as logging_ansi_terminal_init  # 1s
 
 from partcad.logging_ansi_terminal import init as logging_ansi_terminal_init  # 1s
 from partcad_cli.click.loader import Loader
 
+transaction = None
```

Comment:
- [ ] I will use Click's Context instead of global var to pass transaction to sub commands.